nextflow.config

```diff
@@ -27,23 +27,25 @@ params {
     annotation_prodigal_forcenonsd          = false
 
     annotation_bakta_db                     = null
+    annotation_bakta_db_light               = null
```

Comment:
I am a bit confused about the new parameter: 
`--annotation_bakta_db` expects a path to a local DB (might be full or light, depends on what the user downloaded himself)
while `--annotation_bakta_db_light` determines in the modules.config, if in `BAKTA_BAKTADBDOWNLOAD` the argument `--type` will be set to `full` or `light`:
```params.annotation_bakta_db_light ? '--type light' : '--type full' ```
Shouldn't this then be a boolean parameter with default `false` instead of `null`? If no path to an existing baktaDB ist given via `--annotation_bakta_db`, then the full bakta_db is downloaded unless `annotation_bakta_db_light` was set to `true`.
Do we want the `light` or the `full` version to be default when downloading?

Reply:
Ahh yes, that's why reviews are important :D The new parameter should of course be boolean.
I remember my reason for having `null` was that there was an issue with `nf-core schema build` when putting `false` as default, but forgot to check/fix that.
Default: I would go for `full` DB, but can test if there are big differences to the `light` DB in the output. If not, let's use `light` of course.

Reply:
Maybe consider renaming `--annotation_bakta_db` to make clear that this is for a local installation. Something like `--annotation_bakta_db_path` or `--annotation_bakta_db_local`?
I agree with the `full` db as default. Then the parameter `--annotation_bakta_db_light` can just stay as it is.

Reply:
I agree, we should disambiguate, and default should be set to a boolean not `null`. Another suggestion:

```
--annotation_bakta_downloadlightdb
--annotation_bakta_localdbpath
```

Or something?